lib/modules/apostrophe-pieces-submit-widgets/index.js

```diff
@@ -0,0 +1,93 @@
+var async = require('async');
+var _ = require('lodash');
```

Comment:
@Cleop could you add these as dependencies in the `package.json`.
These are currently available as they are dependencies of another `node_module`.
If you use a dep in our codebase (outside of `node_modules`), you should have it listed as a dependency.